pymc3/distributions/dist_math.py

```diff
@@ -108,17 +108,17 @@ def normal_lccdf(mu, sigma, x):
     )
 
 
-def sd2rho(sd):
+def sigma2rho(sigma):
     """
-    `sd -> rho` theano converter
-    :math:`mu + sd*e = mu + log(1+exp(rho))*e`"""
-    return tt.log(tt.exp(tt.abs_(sd)) - 1.)
+    `sigma -> rho` theano converter
+    :math:`mu + sigma*e = mu + log(1+exp(rho))*e`"""
+    return tt.log(tt.exp(tt.abs_(sigma)) - 1.)
 
 
-def rho2sd(rho):
+def rho2sigma(rho):
```

Comment:
Do not forget to run
https://github.com/pymc-devs/pymc3/blob/c496378db707f402c89add9b00ea70015ecab50c/docs/source/notebooks/convolutional_vae_keras_advi.ipynb

Reply:
Also a backward compat function warning about deprecation is needed